src/iidm/converter/SimpleAnonymizer.cpp

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <powsybl/iidm/converter/SimpleAnonymizer.hpp>
+
+#include <istream>
+#include <ostream>
+
+#include <boost/algorithm/string.hpp>
+
+#include <powsybl/PowsyblException.hpp>
+#include <powsybl/stdcxx/format.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace converter {
+
+Country SimpleAnonymizer::anonymizeCountry(const Country& country) {
+    return country;
+}
+
+std::string SimpleAnonymizer::anonymizeString(const std::string& str) {
+    auto it = m_mapping.find(str);
+    if (it != m_mapping.end()) {
+        return it->second;
+    }
+    auto status = m_mapping.emplace(str, getAlpha(m_mapping.size() + 1));
+    m_reverseMapping[status.first->second] = str;
+    return status.first->second;
+}
```

Comment:
I'm not sure how empty strings are handled, but in the java version it's a special case.

Reply:
You're right, I forgot a special case to avoid adding an entry in the map...